.changeset/happy-donuts-taste.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': minor
+---
+
+Adds a new command `astro info`, useful for sharing debugging information about your current environment when you need help!
```

Comment:
```suggestion
Environment info in your CLI

Adds a new command `astro info`, useful for sharing debugging information about your current environment when you need help!
```
Just adding something that will become a heading here, since I think this will end up in the blog post draft, and the existing line would be too long.

Reply:
After discussing with Chris, I'm not *sure* about updating the changeset here! :sweat_smile: 

This is a good changeset for the changelog, but not for the blogpost. So, I'm not sure whether to do any optimizing here at all, since there's no good choice.

As a default, I'll NOT suggest headings and stuff, only edit for typos etc., and let Matthew battle that when it comes blog post time! In which case, this one is fine as is!

Reply:
No bother :) fee free to merge any suggestion you deem right